python/MoltresXS.py

```diff
@@ -0,0 +1,280 @@
+"""
+     Python Function that takes homoginezed cross sections and prepares them for MOLTRES
```

Comment:
spelling of "homogenize". Also, no need to capitalize Moltres. Seems to me that nuclear engineers like to yell all the reactor physics code they write. :smile: